CHANGELOG.md

```diff
@@ -1,20 +1,36 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-This project adheres to the Node default version scheme, meaning It's safe to use with the caret ^ dependency definition.
-The standard for the caret can [be found here](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004).
-Breaking changes result in a different major. UI changes that might break customizations on top of the sdk, will be treated as breaking changes too.
+
+This change log file is based on best practices from [Keep a CHANGELOG](http://keepachangelog.com/).  
+This project adheres to the Node default version scheme, meaning it is safe to use with the caret ^ dependency definition. The standard for the caret can [be found here](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004).  
```

Comment:
I think the line on Node version scheme has too much detail. I would just say `This project adheres to the Node default version scheme.` and link to it. I would also put it below Semver